### ISSUE TITLE
Issue with sniping

### ISSUE THREAD

**Issue by maithanhtan** [PocketMobsters/MobBotTemp#128]
When i using and teleport seem that it have error. 
Do not bring back to the old location to bot again
![image](https://cloud.githubusercontent.com/assets/6257646/17769100/4c45efde-6561-11e6-83ea-e54e594178af.png)

**Comment by tangdami** [comment]
Yeah, I submited this as well. https://github.com/PocketMobsters/MobBotTemp/issues/117
